src/Parser/Priest/Shadow/Modules/Items/ZenkaramIridisAnadem.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SCHOOLS from 'common/MAGIC_SCHOOLS';
+import { formatNumber } from 'common/format';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Analyzer from 'Parser/Core/Analyzer';
+
+const ZENKARAM_VOIDFORM_DAMAGE_INCREASE = 0.23;
+const DEFAULT_VOIDFORM_DAMAGE_INCREASE = 0.20;
+
+class ZenkaramIridisAnadem extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDamage = 0;
+  bonusHealing = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasHead(ITEMS.ZENKARAM_IRIDIS_ANADEM.id);
+  }
+
+  on_byPlayer_damage(event){
+    if(
+      this.combatants.selected.hasBuff(SPELLS.VOIDFORM_BUFF.id) &&
+      event.ability.type === SCHOOLS.ids.SHADOW){
+      this.bonusDamage += event.amount -
```

Comment:
The recommended approach for calculating these kind of damage bonuses is to use `calculateEffectiveDamage`. For example this is missing absorbed damage which more often than not is important in fights.

Reply:
Ah, I will get it changed.